.github/workflows/test-package.yml

```diff
@@ -14,6 +14,9 @@ on:
       stylelint-version-override:
         type: string
         description: The Stylelint version override
+      working-directory-is-workspace:
```

Comment:
npm workspaces often assume that `npm install` and other commands are ran from the project root.